### ISSUE TITLE
TypeError: __init__() got an unexpected keyword argument 'is_target'

### ISSUE THREAD

**Issue by ZoranPandovski** [mindsdb/lightwood#294]
Try to train the model with [auto_insurance](https://github.com/mindsdb/benchmarks/blob/master/benchmarks/datasets/automobile_insurance/data.csv) dataset.
```
  File "mindsdb_acc.py", line 22, in <module>
    print(run())
  File "mindsdb_acc.py", line 9, in run
    mdb.learn(from_data='Auto_Insurance_Claims_Sample.csv', to_predict='Claim Amount')
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/controllers/predictor.py", line 283, in learn
    logger=self.log
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/controllers/transaction.py", line 53, in __init__
    self.run()
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/controllers/transaction.py", line 226, in run
    self._run()
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/controllers/transaction.py", line 222, in _run
    raise e
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/controllers/transaction.py", line 207, in _run
    self._call_phase_module(module_name='ModelInterface', mode='train')
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/controllers/transaction.py", line 164, in _call_phase_module
    return module(self.session, self)(**kwargs)
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/phases/base_module.py", line 53, in __call__
    ret = self.run(**kwargs)
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/phases/model_interface/model_interface.py", line 31, in run
    self.transaction.model_backend.train()
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/mindsdb_native/libs/backends/lightwood.py", line 302, in train
    self.predictor.learn(from_data=train_df, test_data=test_df)
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/lightwood/api/predictor.py", line 111, in learn
    train_ds = DataSource(from_data, self.config)
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/lightwood/api/data_source.py", line 104, in __init__
    self.encoders = self._prepare_encoders()
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/lightwood/api/data_source.py", line 374, in _prepare_encoders
    training_data=input_encoder_training_data)
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/lightwood/api/data_source.py", line 316, in _prepare_column_encoder
    is_target=is_target
  File "/home/zoran/MyProjects/mindsdb-examples/mdb/lib/python3.7/site-packages/lightwood/api/data_source.py", line 297, in _make_column_encoder
    encoder_instance = encoder_class(is_target=is_target)
TypeError: __init__() got an unexpected keyword argument 'is_target'
```

**Comment by paxcema** [comment]
As of release 0.44 this shouldn't be an issue, if you can confirm @ZoranPandovski that'd be great.

**Comment by ZoranPandovski** [comment]
@paxcema This is fixed with a 0.44.0 version. We will update the mindsdb_native to that version.
